src/main/java/com/tterrag/k9/mappings/official/OfficialMapping.java

```diff
@@ -116,7 +133,12 @@ public String getDesc(NameType name) {
     @Override
     public String getIntermediate() {
         if (intermediate == null) {
-            if (type == MappingType.CLASS) {
+            // Should help with random issues
+            mappedDesc.clear();
```

Comment:
This is not a good idea, what's the point of the cache then?

Reply:
The cache is cleared when resolving getIntermediate to fix lingering issues. After that, it stays cached. This is only called once when the intermediate is first requested.